src/backend/joanie/core/admin.py

```diff
@@ -92,6 +101,61 @@ class ProductAdmin(TranslatableAdmin):
 
     inlines = (ProductCourseRelationInline,)
     readonly_fields = ("related_courses",)
+    actions = ("generate_certificate",)
+
+    @admin.action(description=_("Generate certificate"))
+    def generate_certificate(self, request, queryset):
```

Comment:
Could the code in this method be factorized with the management command?

Reply:
Yes you are right. This function has been written before the management command and that's currently a residue. As we discussed last week, I'll try to allow to launch this admin action for a couple `product` & `course`.